docker/README.md

```diff
@@ -58,7 +76,7 @@ docker run -it\
     --env ZK_URL=my-zookeeper:2181 \
     --name "bookie1" \
     --hostname "bookie1" \
-    bookkeeper
+    apache/bookkeeper:4.5.0
```

Comment:
Tag 4.5.0 is not needed. If you not provide it, docker will use "latest" (that we will always make it pointing to the latest released version, and it is probably the best choice for a new user). If we besides "hardcode" this tag in readme, we need to update it at every release.

Reply:
ok, thanks, I will change it soon.